### ISSUE TITLE
Create a call to action button

### ISSUE THREAD

**Issue by mariagould** [ror-community/ror-site#53]
ROR website editors should be able to add a call to action button element to pages on the ROR site. It should look like the call to action button that already exists on the homepage:

![Screen Shot 2022-07-07 at 2 35 21 PM](https://user-images.githubusercontent.com/45110557/177857560-e99b5795-869b-4497-a87f-9d67884864f3.png)

There should be a gray option like on the homepage, and another color option that uses the ROR teal (#53baa1) and black link text.

**Comment by cazinc** [comment]
We have created a shortcode for this in the Summer-2022-updates branch, which allows for some variation in colour and icon. Have a look at http://localhost:1313/test/ locally to see, and look at `../content/test/_index.md` to read about how to use the new shortcode. Screenshot attached here for ref too. 

![screencapture-localhost-1313-test-2022-08-07-18_36_00](https://user-images.githubusercontent.com/3863704/183303668-84c265d3-209f-4b4d-beea-faec9b3fbe5d.png)

**Comment by cazinc** [comment]
We have created a shortcode for this in the Summer-2022-updates branch, which allows for some variation in colour. Have a look at http://localhost:1313/test/ locally to see, and look at ../content/test/_index.md to read about how to use the new shortcode. 

![screencapture-localhost-1313-test-2022-08-07-18_36_00](https://user-images.githubusercontent.com/3863704/183303921-8bb60fa3-3428-46ec-8543-c739446148d2.png)
Screenshot attached here for ref too.

**Comment by mariagould** [comment]
Buttons look great @cazinc. Quick question: can a button be center-aligned as well?

**Comment by cazinc** [comment]
Hi @mariagould it can be, yes. There are a few options here. We can provide you with a shortcode that you could wrap around anything you want centered, so just a button or really anything bigger too. Or we could rework the buttons to have a way to specify them as centred optionally. What's your pref?

**Comment by mariagould** [comment]
Thanks @cazinc - a shortcode that allows anything to be centered would be great! But if the latter option is the easier one to implement, happy to go with that for now :)

**Comment by cazinc** [comment]
Great, we've added a new `centered` shortcode now that you can use just about anywhere @mariagould. Committed to the Summer-2022-updates branch and you can see an example of it in use right at the bottom of http://localhost:1313/test/.

**Comment by mariagould** [comment]
Documentation feedback from discussion earlier today: We'd like to avoid the rusty button color if possible. Perhaps something closer to the ROR orange? Per discussion, we talked about adjusting the teal and red/orange buttons to a slightly more prominent hue.

**Comment by cazinc** [comment]
We have made a few changes here @mariagould:

1. Rusty red colour removed in favour of the standard ROR orange/red;
2. Button hover effect changed a bit: now buttons lift but retain same colour (the rusty red was just a product of trying to find an "off" colour for the previous hover effect);
3. We have replaced the green buttons with the mustard. 

I think actually this limited use of the mustard for buttons works well as it is more prominent than the green, especially due to its limited use in the site so far.
